Dockerfile

```diff
@@ -66,8 +66,8 @@ COPY ./*.json /openedx/edx-platform/
 COPY ./openedx.yaml /openedx/edx-platform/
 COPY ./codecov.yml /openedx/edx-platform/
 COPY ./circle.yml /openedx/edx-platform/
-COPY ./setup.cfg /openedx/edx-platform/
-COPY ./setup.py /openedx/edx-platform/
+COPY ./setup.cfg ./setup.py /openedx/edx-platform/
+# COPY ./setup.py /openedx/edx-platform/
```

Comment:
Do we need this still?

Reply:
With this system it's hard to tell, but it's nice to have the setup.py for local debugging in a lot of cases.